Integrations/integration-EWSMailSender.yml

```diff
@@ -141,10 +143,20 @@ script:
         cc = cc.split(",") if cc is not None else None
         to = to.split(",") if to is not None else None
 
-        file_entries_for_attachments = attachIDs.split(",") if attachIDs else []
-        attachments_names = attachNames.split(",") if attachNames else []
-        if len(file_entries_for_attachments) != len(attachments_names):
-            raise Exception("attachIDs and attachNames lists should be the same length")
+        file_entries_for_attachments = []
+        attachments_names = []
+        if attachIDs:
+            file_entries_for_attachments = attachIDs.split(",")
+            if attachNames:
+                attachments_names = attachNames.split(",")
+            else:
+                for att_id in file_entries_for_attachments:
+                    att_name = demisto.dt(demisto.context(), "File(val.EntryID === '%s').Name" % att_id)
```

Comment:
Is this the right way to do this?
You also get the name with the `getFilePath` response... the file might not be in the context...